BootloaderCommonPkg/Library/LiteVariableLib/LiteVariableLib.inf

```diff
@@ -38,7 +38,7 @@
   HobLib
 
 [Guids]
-
+  gZeroGuid
```

Comment:
Is it mandatory to specify ZeroGuid if NULL?

Reply:
That's the implementation choice to simplify the caller.